store/Cargo.toml

```diff
@@ -27,6 +27,7 @@ miden-node-utils = { path = "../utils" }
 miden_objects = { workspace = true }
 once_cell = { version = "1.18.0" }
 prost = { version = "0.12" }
+rusqlite = { version = "0.29", features = ["array", "buildtime_bindgen"] }
```

Comment:
`array` is needed to support passing multiple values to the cursor. This is used to allow querying for tag/nullifier prefixies and account ids.

The `rusqlite` library supports multiple versions of the `sqlite`. It can use the system's library, it can statically link `sqlite` to the final binary, and it can use a patched version of `sqlite` with additional encryption features. To speed up compilation, the library is distributed with ffi generated for a conservative version of `sqlite` that can be used with any of the aforementioned distributions. Unfortunately it doesn't support the `array` extension. So  we have to generate the ffi.